src/components/Screens/PlanexWebview.tsx

```diff
@@ -129,7 +130,13 @@ function PlanexWebview(props: Props) {
         showControls={props.currentGroup !== undefined}
         incognito={true}
       />
-      {!props.currentGroup ? (
+      {props.isPlanexDown ? (
+        <ErrorView
```

Comment:
Current behavior is to wait until webview has loaded (showing loading animation) before showing unavailable error message. See `getRenderLoading` in `src/components/Screens/PlanexWebview.tsx`

Reply:
Isn't the loading animation already  showed until the error/calendar is shown ?

Are you talking about this ?
https://github.com/ClubInfoInsaT/application-amicale/blob/aaf72d9122154fb6c822801fe24b33403c052106/src/components/Screens/RequestScreen.tsx#L29